src/Sarif/NotYetAutoGenerated/Address.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// A physical or virtual address, or a range of addresses, in an 'addressable region' (memory or a binary file).
     /// </summary>
     [DataContract]
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "2.1.0.0")]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "2.3.0.0")]
     public partial class Address : PropertyBagHolder, ISarifNode
```

Comment:
>partial [](http://example.com/codeflow?start=11&length=7)

so do you know for sure not that every file in notyetautogenerated contains other code changes that need to be preserved? or can we delete any of these?

Reply:
Yes I did have compared that every file in the notyetautogenerated folder have difference between the raw auto generated one, when work on these I believe some tests are needed, I think for the separate change.